### ISSUE TITLE
A quick question about choosing decoder

### ISSUE THREAD

**Issue by HelloWorldLTY** [mtang724/NWR-GAE#5]
![image](https://user-images.githubusercontent.com/43333475/223726228-656ac4c0-5dc0-4de6-94dd-a8479bf55e10.png)
Hi, your paper is very interesting. Could I ask one question related to decoding degree and initial feature? I think it is not inituitive to choose these two approaches, could you please talk more about motivation? Moreover, why you choose to decode the initial feature rather than the original input x_v? I think the later choice is more common used in auto-encoder area. Thanks a lot.

**Comment by mtang724** [comment]
Hi thank you for your great questions. For the first question, the motivation behind degree decoder is we do not know how many neighbors a node have when reconstructing it's neighbors feature. Moreover, because we only sampled a small range of neighbors to do the reconstruction since the computation cost is high for the reconstruction process. The decoder of the init feature is to reconstruct the neighborhood information layer by layer (or directly to the first layer for simplicity). For the second question, in practice, the original node features X could be very high-dimensional and reconstructing them directly may introduce a lot of noise into the node representations. Instead, we may first map X into a latent space with pair norm. Hope I answered your question, and hope you are having a great week.
